lib/workers/repository/dependency-dashboard.ts

```diff
@@ -13,11 +13,49 @@ import { PackageFiles } from './package-files';
 interface DependencyDashboard {
   dependencyDashboardChecks: Record<string, string>;
   dependencyDashboardRebaseAllOpen: boolean;
+  dependencyDashboardAllPending: boolean;
+  dependencyDashboardAllRateLimited: boolean;
+}
+
+function checkApproveAllPendingPR(issueBody: string): string {
+  const checkApproveAllPendingPR =
+    ' - \\[x\\] <!-- approve-all-pending-prs -->';
+  const checkedApproveAll = issueBody.match(
+    regEx(checkApproveAllPendingPR, 'g')
+  );
+  let newIssueBody = issueBody;
+  if (checkedApproveAll?.length) {
```

Comment:
```suggestion
  let newIssueBody = issueBody;
  if (issueBody.test(regEx(checkApproveAllPendingPR) {
```

if you need to only check for existence, use [test](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test).

it is also more preferment